torch/csrc/onnx/init.cpp

```diff
@@ -235,9 +235,10 @@ void initONNXBindings(PyObject* module) {
 
   m.def(
       "_check_onnx_proto",
-      [](const std::string& proto_string, bool full_check) {
-        check_onnx_proto(proto_string, full_check);
-      },
+      ::torch::wrap_pybind_function(
```

Comment:
Is this really needed? Many bindings in this file do not use. Try to understand what `::torch::wrap_pybind_function` does before start using it. If it is not needed, the lesser code, the better.

Reply:
Yes. See https://github.com/pytorch/pytorch/pull/78441

Reply:
`::torch::wrap_pybind_function` helps to wrap the error into warning which help the PR achieve the goal of "not error out the exporter, but give the warning of why this model doesn't pass `strict_mode` shape type inference"